index.html

```diff
@@ -1,226 +1,174 @@
+      <a href="https://github.com/naanmohammed">LET'S CONNECT</a>
+      <ul>
+        <li><a href="www.twitter.com/_naanahmed" target="_blank"><img src="images/twitter.jpg"></a></li>
+        <li><a href="www.linkedin.com/in/naanahmed" target="_blank"><img src="images/i.jpg"><img src ="images/n.jpg"></a></li>
+        <li><a href="www.medium.com/naanahmed" target="_blank"><img src="images/medium.jpg"></a></li>
+        <li><a href="www.github.com/naanmohammed" target="_blank"><img src="images/github.jpg"></a></li>
+        <li><a href="www.reddit.com/naanahmed" target="_blank"><img src="images/reddit.jpg"></a></li>
+      </ul>
+      <button>Get my resume</button>
     </div>
-    <section class="purple">
-    <div class="contact_me">
-        <h2>Contact me</h2>
-        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-
-        <form action="https://formspree.io/f/xvonabyg"
-        method="POST">
-            <ul>
-                <li><input value="" placeholder="Enter your name" id="name" type="text" name="user_name" maxlength="30" required></li>
-                <li><input value="" placeholder="Enter your email" id="email" type="email" name="email" required></li>
-            </ul>
-            <textarea value="" id="message" placeholder="Write your message here" type="text" name="message" maxlength="500" required></textarea>
-            <button>Get in touch</button>
-        </form>
+    <div class="languages">
+      <div class="language_header">
+        <h3>Languages</h3>
+        <p class="down_arrow"></p>
+      </div>
+      <ul class="language_list">
+        <li><img src="images/javascript.jpg"><p>Javascript</p></li>
+        <li><img src="images/html.jpg"><p>HTML</p></li>
+        <li><img src="images/css.jpg"><p>CSS</p></li>
+      </ul>
+      <div class="skill_frameworks">
+        <h3>Frameworks</h3>
+        <p class="icon_right"></p>
+      </div>
+      <div class="skill_frameworks">
+        <h3>Skills</h3>
+        <p class="icon_right"></p>
+      </div>
     </div>
-    </section>
+  </section>
+  <section class="contact_me">
+    <h2>Contact me</h2>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <form action="https://formspree.io/f/xvonabyg">
+      <ul>
+        <li><input type="text" placeholder="Full Name" id="user_name" required maxlength="30"></li>
+        <li><input type="email" id="email" name="user_mail" placeholder="Email Address" pattern="/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+.[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-])*$/" required></li>
+        <li><textarea name="user_message" id="textarea" cols="30" rows="10" placeholder="Write your message here" required maxlength="500"></textarea></li>
+      </ul>
```

Comment:
[OPTIONAL] In order to follow best practices, I recommend excluding the `ul` `li` tags from the form and le the input stand alone